server/base/src/main/java/org/apache/accumulo/server/util/DeleteZooInstance.java

```diff
@@ -103,6 +112,19 @@ private static void cleanAllOld(ServerContext context, final ZooReaderWriter zk)
     }
   }
 
+  private static boolean checkCurrentInstance(ServerContext context, String instanceName,
+      String instanceId) {
+    if (instanceId.equals(context.getInstanceID().canonical())) {
+      String prompt = String.valueOf(
+          System.console().readLine("Warning: This is the current instance, are you sure? Y/n: "));
+      if (prompt == null || !prompt.equals("Y")) {
```

Comment:
I know in other areas (like when deleting a table or namespace with the shell) we use (yes|no) and we also ignore cases. Not really sure how consistent we are with that though. At least ignoring the case would be a reasonable expectation from the user's viewpoint.

Reply:
Table deletion verification is done [here](https://github.com/apache/accumulo/blob/main/shell/src/main/java/org/apache/accumulo/shell/commands/TableOperation.java#L90), as an example.

Reply:
I agree 100% and will change it. I didn't know if we had other examples of confirm prompts but I should have searched because I agree that it should be made consistent and match if we already have a normal pattern for prompting the user for yes/no.